spec/class.dd

```diff
@@ -371,6 +371,9 @@ $(GNAME Constructor):
         }
         ------
 
+        $(P The compiler generate a $(D __ctor) alias that matches with
+        constructors definitions.)
```

Comment:
```suggestion
        constructor definitions.)
```

Reply:
I don't really understand what you mean by this sentence. Could you please elaborate?